### PR TITLE
Created farmRouter, productRouter

### PR DESCRIPTION
**Pull request by sudarshankannan**

### CODE REVIEW

File: routes/farmRouter.js [hunk 1]
```diff
@@ -1,27 +1,57 @@
 const router = require('express').Router();
-const farm = require('../models/Farm');
+const res = require('express/lib/response');
```
Comment (Ahrorus) [inline routes/farmRouter.js:2]: we shouldn't be needing that

File: routes/farmRouter.js [hunk 2]
```diff
@@ -1,27 +1,57 @@
...
 router.post('/', async (req, res) => {
     try {
-        
+        const farm = new Farm({
+            name: req.body.name,
+            location: req.body.location
+        });
     } catch (err) {
-        console.log("hello");
+        res.json({message: err});
```
Comment (Ahrorus) [inline routes/farmRouter.js]: better yet if you do `res.status(500).send('Server error. Unable to create new farm.');`

File: routes/farmRouter.js [hunk 3]
```diff
@@ -1,27 +1,57 @@
...
 });
 
 router.get('/:farmId', async (req, res) => {
     try {
-        const farms = await farm.findById(req.params.farmId);
+        const farm = await Farm.findById(req.params.farmId);
+        res.json(farm);
     } catch (err) {
-        
+        res.json({message: err});
```
Comment (Ahrorus) [inline routes/farmRouter.js]: better yet if you do res.status(404).send('Farm not found.');

File: routes/farmRouter.js [hunk 4]
```diff
@@ -1,27 +1,57 @@
...
     }
 });
 
 router.post('/', async (req, res) => {
     try {
-        
+        const farm = new Farm({
```
Comment (Ahrorus) [inline routes/farmRouter.js:28]: I think the farm should have other data too, like `belongsTo` or `userId`

File: routes/farmRouter.js [hunk 5]
```diff
@@ -1,27 +1,57 @@
...
+            {_id: req.params.farmId}, 
+            {$set: {
+                name: req.body.name,
+                location: req.body.location
+            }}
+        );
+        res.json(updatedFarm);
+    }
+    catch(err){
+        res.json({message: err});
```
Comment (Ahrorus) [inline routes/farmRouter.js]: Same as before, can send 500 status code

File: routes/farmRouter.js [hunk 6]
```diff
@@ -1,27 +1,57 @@
...
+const Farm = require('../models/Farm');
 
 router.get('/', async (req, res) => {
     try {
-        const farms = await farm.find();
-        res.json();
+        const farms = await Farm.find();
+        res.json(farms);
     } catch (err) {
         res.json({message: err})
```
Comment (Ahrorus) [inline routes/farmRouter.js]: Can also attach 500 status code, Internal server error

**Review by Ahrorus — COMMENTED** [review]
Could also attach similar status codes for userRouter, similar to farmRouter

**Comment by Ahrorus** [comment]
Also, make sure to carefully resolve the conflicts before merging

File: routes/farmRouter.js (hunk 1 above)
Comment (sudarshankannan) [reply routes/farmRouter.js:2]: Pretty sure we do... since Farm and Farm router aren't in the same sub-folders

**Pull request merged**